docs/experiments-plus/sequential-testing.md

```diff
@@ -0,0 +1,74 @@
+
+Click on Edit at the top of the metrics section in Pulse to toggle Sequential Testing on/off.
+
+![image](https://user-images.githubusercontent.com/90343952/191135447-5e094892-49e5-485e-8186-18732888662c.png)
+
+When enabled, an adjustment is automatically applied to results calculated before the target completion date of the experiment. 
+
+![image](https://user-images.githubusercontent.com/90343952/191135645-0042dced-3e8f-479f-8f63-c814dfbd4923.png)
+
+The dashed line represents the expanded confidence interval resulting from the adjsutment.  The solid bar is the standard confidence interval computed without any adjustments.  If the adjusted confience interval overalps with zero, this means the metric delta is not stat-sig at the moment, and the experiment should continue its course as planned. 
+
+Hover over a metric and click **View Details** to review the progression of the sequential test.  
+
+(screen shot coming)
+
+The **Sequential Testing Z-Statistic** time series contains the following information for a metric:
+
+* **Efficacy Boundaries** (solid red and green curves): The thresholds for positive and negative statistical significance.  These start out high, signifying the increased confidence needed for making an early decision. When the target duration is reached, they converge to the standard Z-score for the selected significance level (dashed lines).
+* **Measurement Z-score** (dots): These are the Z-scores computed each day for the test vs. control comparisson.  A Z-score higher than the upper efficacy boundary is stat-sig positive.  One lower than the bottom boundary is stat-sig negative.  
+
+
+## Statsig's Implementation of Sequential Testing
+
+We use an adjustment factor *q<sub>n</sub>* that's determined by the number of days *n* the experiment has been running:
+
+![image](https://user-images.githubusercontent.com/90343952/191127696-c8cbbf6f-8757-439e-86df-c7d7dd13ef36.png)
+
+When the target duration is reached, *q<sub>n</sub> = 1* and no more adjustments are applied.  This method has 2 benefits:
+* Simplicity: The calculation of the adjustment factor is easy to understand.  It also satisifes the intuitive expectation that the significance threshold be higher early on.
+* Power: When the target duration is reached, the efficacy boundary converges with the standard Z-score for the selected significance level.  Therefore, there is no loss in statistical power when doing a metrics readout at the conclusion of the pre-determined experiment duration. We selected this approach because we believe the primary value of sequential testing is to provide higher confidence when making early decisions based on unexpected metric movements, such as ending an experiment early due to a large regression.  However, in most cases it's best make a decision based on the complete set of relevant metrics at the end of the experiment, without any adjsutments that reduce power. 
+
+### Efficacy Boundary and Z-value Calculation
```

Comment:
This section looks good to me.  But I did want to know if you purposely used the term "Z-value" instead of "Z-score" to distinguish an adjusted value from the standard one.